src/JuMP.jl

```diff
@@ -59,7 +60,7 @@ MOIU.@model(_MOIModel,
              MOI.RootDetConeTriangle, MOI.RootDetConeSquare,
              MOI.LogDetConeTriangle, MOI.LogDetConeSquare),
             (),
-            (MOI.SingleVariable,),
+            (),
```

Comment:
You can get rid of this and use `MOIU.Model` instead of `_MOIModel`

Reply:
Great, that works.